### ISSUE TITLE
Severe errors about subsurface with outward facing angle significantly different from base surface should always report regardless of warnings level

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5016]
Helpdesk ticket 10613
User notes that skylights vertices were reversed so the skylights were upside down, but no error messages reported the problem.  There are error messages like this, but they only show when DisplayExtraWarnings or DisplayAllWarnings has been requested.

```
** Severe ** GetSurfaceData: Outward facing angle [0.000] of 
subsurface="SKYLIGHT 1" significantly different than
** ~~~ ** ..facing angle [180.000] of base surface=ROOFSLAB 
SLAB FOR FLOOR 2(2)
** ~~~ ** ..surface class of base surface=Roof
** Severe ** GetSurfaceData: Outward facing angle [0.000] of 
subsurface="SKYLIGHT 1(1)" significantly different than
** ~~~ ** ..facing angle [180.000] of base surface=ROOFSLAB 
SLAB FOR FLOOR 2(2)
** ~~~ ** ..surface class of base surface=Roof
```

These errors are severe enough that they should always be reported, regardless of the warnings level, and possibly even be considered fatal, since a backwards window will produce bad results, whether on a wall or on a roof.

**Comment by mjwitte** [comment]
Not sure what the threshold is for this, but I would say anything that differs by more than 90 degrees should produce a fatal error message.  And, say, 45-90 degrees can be severe?  Not sure.

**Comment by rglazer2** [comment]
@mjwitte Currently the code is comparing the subsurface angle with the surface angle and printing a severe error(when all warnings are requested) whenever the two angles do not match. Do you think it should remain that way and just always print the error, or should I add what you suggested earlier about different levels of severity based on the angle difference. 

This is occurring in SurfaceGeometry.cc:1155

**Comment by mjwitte** [comment]
@rglazer2 It's interesting that the block above line 1155 is checking the surface azimuths on subsurfaces that are on a horizontal surface (roof or floor).  This should really be checking tilt.  If you look further down at line 1353 you'll see there is a generic message that reports when DispalyExtraWarnings is true.  I'd suggest a similar generic message in the earlier block, and I'd also change the test to add a check for the difference in tilt being >30.  I guess you should keep the azimuth test but set it up so if either one fails then the error is triggered.  I'll need to think about some other test cases to cover the different combinations.

**Comment by Myoldmopar** [comment]
Fixed via #5345
